docs/changes.rst

```diff
@@ -3,6 +3,16 @@ Changelog
 
 14.3.1 (unreleased)
 -------------------
+
+- HtmlToWord: Fix problem with different documents for each report compilation.
+  This involves an API change: Instantiate HtmlToWord compiler freshly for each
```

Comment:
If we want this to be the customer-facing change log then I would leave out the technical information.

Reply:
:+1: